app/controllers/nonsipp/shares/SharesCYAController.scala

```diff
@@ -728,7 +742,13 @@ object SharesCYAController {
                   SummaryAction(
                     "site.change",
                     routes.SharesFromConnectedPartyController.onPageLoad(srn, index, mode).url
-                  ).withVisuallyHiddenContent("sharesCYA.section3.sharesFromConnectedParty.hidden")
+                  ).withVisuallyHiddenContent(
+                    Message(
+                      "sharesCYA.section3.sharesFromConnectedParty.hidden",
```

Comment:
Can you combine this logic with the previous one to simplify here ? 

Like this :

```
sharesFromConnectedParty.flatMap { connectedParty =>
          holdShares match {
            case SchemeHoldShare.Transfer =>
              Some(
                CheckYourAnswersRowViewModel(
                  Message("sharesCYA.sectionTransfer.sharesFromConnectedParty", companyNameRelatedShares),
                  if (connectedParty) "site.yes" else "site.no"
                ).withAction(
                  SummaryAction(
                    "site.change",
                    routes.SharesFromConnectedPartyController.onPageLoad(srn, index, mode).url
                  ).withVisuallyHiddenContent(
                    Message(
                      "sharesCYA.sectionTransfer.sharesFromConnectedParty.hidden",
                      companyNameRelatedShares,
                      if (connectedParty) "site.yes" else "site.no"
                    )
                  )
                )
              )
            case SchemeHoldShare.Contribution =>
              Some(
                CheckYourAnswersRowViewModel(
                  Message("sharesCYA.sectionContribution.sharesFromConnectedParty", companyNameRelatedShares),
                  if (connectedParty) "site.yes" else "site.no"
                ).withAction(
                  SummaryAction(
                    "site.change",
                    routes.SharesFromConnectedPartyController.onPageLoad(srn, index, mode).url
                  ).withVisuallyHiddenContent(
                    Message(
                      "sharesCYA.sectionContribution.sharesFromConnectedParty.hidden",
                      companyNameRelatedShares,
                      if (connectedParty) "site.yes" else "site.no"
                    )
                  )
                )
              )
            case _ => None
          }
        }
``